### PR TITLE
App topology

### PR DESCRIPTION
**Pull request by jeswanke**

### CODE REVIEW

File: frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/helpers/constants.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+module.exports.MCM_CLUSTERS_SERVICES_ACTIONS = {}
+
+module.exports.DOC_PATH_BASE =
+    'https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.4/html/applications/managing-applications'
+
+module.exports.DOC_LINKS = {
+    HOME: `${module.exports.DOC_PATH_BASE}`,
+}
```
Comment (KevinFCormier) [inline frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/helpers/constants.js:96]: Are all of these constants actually used? If not, can we remove what isn't please?

File: frontend/src/components/Topology/Topology.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+            this.state.isLoaded !== nextState.isLoaded
+        )
+    }
+
+    render() {
+        const { fetchControl = {}, t } = this.props
+        const { isFailed = false } = fetchControl
+
+        if (isFailed) {
+            return <AcmAlert className="persistent" title={t('error.default.description')} variant="danger" isInline />
```
Comment (chenz4027) [inline frontend/src/components/Topology/Topology.js:142]: Because of the translation, should we do t('the text') instead of using the .json files?

Comment (KevinFCormier) [reply frontend/src/components/Topology/Topology.js:142]: We do have a mix currently. It's not required at the moment to do it one way or the other.

Comment (chenz4027) [reply frontend/src/components/Topology/Topology.js:142]: kk sounds good!

**Review by chenz4027 — APPROVED** [review]
lgtm

**Pull request merged**